### PR TITLE
Added commands for starting and stopping a webserver that can be used -- NEEDS TESTING

### PR DESCRIPTION
**Pull request by aweingarten**
Fixes https://github.com/acquia/blt/issues/1564

Changes proposed:
- Add tests:run-server and blt tests:kill-server

### CODE REVIEW

File: src/Robo/Commands/Tests/BehatCommand.php [hunk 1]
```diff
@@ -126,6 +126,12 @@ public function behatDefinitions($options = ['mode' => 'l']) {
 
   /**
    * Launches PHP's internal web server via `drush run-server`.
+   *
+   * @command tests:run-server
+   * @description This initiates a webserver that we can use for behat or other testing.
+   * @validateDrupalIsInstalled
```
Comment (grasmash) [inline src/Robo/Commands/Tests/BehatCommand.php]: I don't think that Drupal needs to be installed in order to run a server.

File: src/Robo/Commands/Tests/BehatCommand.php [hunk 2]
```diff
@@ -126,6 +126,12 @@ public function behatDefinitions($options = ['mode' => 'l']) {
 
   /**
    * Launches PHP's internal web server via `drush run-server`.
+   *
+   * @command tests:run-server
+   * @description This initiates a webserver that we can use for behat or other testing.
```
Comment (grasmash) [inline src/Robo/Commands/Tests/BehatCommand.php]: You can leave this out, the command will inherit `Launches PHP's internal web server via drush run-server`

File: src/Robo/Commands/Tests/BehatCommand.php (hunk 1 above)
Comment (aweingarten) [inline src/Robo/Commands/Tests/BehatCommand.php]: updated

File: src/Robo/Commands/Tests/BehatCommand.php (hunk 2 above)
Comment (aweingarten) [inline src/Robo/Commands/Tests/BehatCommand.php]: updated

File: src/Robo/Commands/Tests/BehatCommand.php [hunk 3]
```diff
@@ -126,6 +126,10 @@ public function behatDefinitions($options = ['mode' => 'l']) {
 
   /**
    * Launches PHP's internal web server via `drush run-server`.
+   *
+   * @command tests:run-server
+   * @validateInsideVm
+   * @executeInDrupalVm
    */
   protected function launchWebServer() {
     if ($this->getConfigValue('behat.run-server')) {
```
Comment (grasmash) [inline src/Robo/Commands/Tests/BehatCommand.php]: This command should probably run regardless of this config value. We'll need to move this check up a method.

**Comment by aweingarten** [comment]
Closing in favor of https://github.com/acquia/blt/pull/1570/files

**Pull request closed without merging**